dora/src/semck/clsdefck.rs

```diff
@@ -88,7 +88,14 @@ impl<'x> ClsDefCheck<'x> {
             AllowSelf::No,
         )
         .unwrap_or(SourceType::Error);
-        self.add_field(f.pos, f.name, ty, f.mutable, f.is_pub);
+        let is_pub = f.annotation_usages.contains(
```

Comment:
It seems we use such code a lot: Let's add a method `contains_pub` to AnnotationUsages.

Reply:
The problem is that we need `VM` to look up the name, so it would need to be done differently (or not in `AnnotationUsages`/dora-parser). What do you think?

Reply:
I guess one could move the methods to the `dora` module. I'll do that, if you don't have a different preference.

Reply:
Sounds good.